### ISSUE TITLE
How to solve the `__hfma2` problem docker error

### ISSUE THREAD

**Issue by fcolecumberri** [oobabooga/text-generation-webui#2128]
I managed to solve the `__hfma2` problem reported https://github.com/oobabooga/text-generation-webui/issues/593 and https://github.com/oobabooga/text-generation-webui/issues/566

To do so, I changed the lines:

https://github.com/oobabooga/text-generation-webui/blob/fd743a02074d58d2515116d558b4800da15bb9ee/docker/Dockerfile#L19-L21

for

```dockerfile
ARG TORCH_CUDA_ARCH_LIST="7.5"
RUN . /build/venv/bin/activate && \
    TORCH_CUDA_ARCH_LIST="7.5" python3 setup_cuda.py bdist_wheel -d .
```

I am 100% sure this is not how the dockerfile should be written, but my experience with docker is essentially non existent, therefore I hope someone with more experience provides the corresponding pull request.

**Comment by Ph0rk0z** [comment]
I think the issue is that the M series GPU doesn't have this function. The GPTQ nvidia kernel has a function that substitutes for it but it doesn't run unless you set the cuda arch.. in docker it is not detected because there is no access to the actual GPU version?

**Comment by fcolecumberri** [comment]
No, the issue is that CUDA arch 3.5 doesn't support that function, and it is not reading the correct TORCH_CUDA_ARCH_LIST from the .env

**Comment by chrisbward** [comment]
Getting this issue on a 3090ti from the Dockerfile

**Comment by chrisbward** [comment]
> I managed to solve the `__hfma2` problem reported #593 and #566
> 
> To do so, I changed the lines:
> 
> https://github.com/oobabooga/text-generation-webui/blob/fd743a02074d58d2515116d558b4800da15bb9ee/docker/Dockerfile#L19-L21
> 
> for
> 
> ```dockerfile
> ARG TORCH_CUDA_ARCH_LIST="7.5"
> RUN . /build/venv/bin/activate && \
>     TORCH_CUDA_ARCH_LIST="7.5" python3 setup_cuda.py bdist_wheel -d .
> ```
> 
> I am 100% sure this is not how the dockerfile should be written, but my experience with docker is essentially non existent, therefore I hope someone with more experience provides the corresponding pull request.

This fixed the issue for me, thanks!
